### PR TITLE
Fixed package names for heron_desktop and warthog_simulator.

### PR DESCRIPTION
**Pull request by tonybaltovski**
Sorry guys, I used dashes instead of underscores.  I re-bloomed, thus the version change.

### CODE REVIEW

**Comment by mikaelarguedas** [comment]
Arg this will need to be reordered alphabetically then

**Comment by tonybaltovski** [comment]
@mikaelarguedas is that something I can do?  They are in alphabetically order currently.

**Comment by mikaelarguedas** [comment]
If you look at the Travis log it should point you to the command you can run locally to rearrange your distro file alphabetically (and print the diff that needs to be applied). I'm responding from a phone so I can't really look at the log myself.

File: indigo/distribution.yaml [hunk 1]
```diff
@@ -16541,7 +16541,7 @@ repositories:
       url: https://github.com/warthog-cpr/warthog.git
       version: indigo-devel
     status: developed
-  warthog-simulator:
+  warthog_simulator:
```
Comment (mikaelarguedas) [inline indigo/distribution.yaml]: This should be after `warthog_desktop`

**Comment by tonybaltovski** [comment]
Thanks for that @mikaelarguedas :beers:

**Review by mikaelarguedas — APPROVED** [review]
:+1:

**Pull request merged**